stdlib_list/base.py

```diff
@@ -58,18 +58,17 @@ def stdlib_list(version: str | None = None) -> list[str]:
 
     data = pkgutil.get_data("stdlib_list", module_list_file).decode()  # type: ignore[union-attr]
 
-    result = [y for y in [x.strip() for x in data.splitlines()] if y]
+    result = [y for x in data.splitlines() if (y := x.strip())]
```

Comment:
Alternative:

```suggestion
    result = list(filter(None, map(str.strip, data.splitlines())))
```

I don't know your views on functional-style, though!